9/.dockerignore

```diff
@@ -0,0 +1,25 @@
+**/.classpath
+**/.dockerignore
+**/.env
+**/.git
+**/.gitignore
+**/.project
+**/.settings
+**/.toolstarget
+**/.vs
+**/.vscode
+**/*.*proj.user
+**/*.dbmdl
+**/*.jfm
+**/azds.yaml
+**/bin
+**/charts
+**/docker-compose*
+**/Dockerfile*
+**/node_modules
+**/npm-debug.log
+**/obj
+**/secrets.dev.yaml
+**/values.dev.yaml
+LICENSE
+README.md
```

Comment:
Not sure we'll need this gitignore. We already have one on the root level: https://github.com/PiotrJustyna/road-to-orleans/pull/202/files

Reply:
Looks like solution 6 also has as gitignore (https://github.com/PiotrJustyna/road-to-orleans/blob/main/6/.dockerignore), creating an issue to remove them from individual solutions.

Reply:
Done: https://github.com/PiotrJustyna/road-to-orleans/issues/203